tests/src/Kernel/Entity/GetGroupContentTest.php

```diff
@@ -82,6 +95,7 @@ public function testBasicGroupReferences() {
     $groups['entity_test'] = EntityTest::create([
       'type' => $bundle,
       'name' => $this->randomString(),
+      'uid' => $this->groupAdmin->id(),
```

Comment:
Getting back to this, in `EntityTest` the user ID field is actually called `user_id` instead of `uid`, so if this is required to make the tests pass then we probably have a bug somewhere.